pkg/lib/operatorstatus/csv_reporter.go

```diff
@@ -98,9 +102,9 @@ func (r *csvStatusReporter) GetNewStatus(existing *configv1.ClusterOperatorStatu
 
 	switch phase {
 	case v1alpha1.CSVPhaseSucceeded:
-		builder.WithAvailable(configv1.ConditionTrue, "")
+		builder.WithAvailable(configv1.ConditionTrue, fmt.Sprintf("ClusterServiceVersion %v/%v is in phase %v", csv.Namespace, csv.Name, csv.Status.Phase), reasonCSVSucceeded)
 	default:
-		builder.WithAvailable(configv1.ConditionFalse, "")
+		builder.WithAvailable(configv1.ConditionFalse, fmt.Sprintf("ClusterServiceVersion %v/%v is in phase %v", csv.Namespace, csv.Name, csv.Status.Phase), reasonCSVNotSucceeded)
```

Comment:
I'd add the CSV's .status.reason and/or .status.message at the end of this message. The person reading this message may have been paged while away from work/home and will appreciate having a clearer picture of what might be wrong before opening their laptop and investigating.

Reply:
Done.